environment.yml

```diff
@@ -1,7 +1,9 @@
 name: shiver
 channels:
   - conda-forge
-  - mantid/label/nightly
+  - mantid
+  - mantid-ornl
```

Comment:
I think you want `mantid-ornl` before `mantid` here so our fork has higher priority